nuxeo-core/nuxeo-core-storage-dbs/src/main/java/org/nuxeo/ecm/core/storage/dbs/DBSSession.java

```diff
@@ -720,11 +744,12 @@ public Document copy(Document source, Document parent, String name) {
         return getDocument(copyState);
     }
 
-    protected String copyRecurse(String sourceId, String parentId, LinkedList<String> ancestorIds, String name) {
+    protected String copyRecurse(String sourceId, String parentId, LinkedList<String> ancestorIds, String name, boolean excludeSpecialChildren) {
```

Comment:
Pass both `boolean excludeRegularChildren, boolean excludeSpecialChildren` here to be clear.